c++/src/kj/compat/http.c++

```diff
@@ -2649,8 +2659,9 @@ private:
     if (recvData.size() > 0) {
       // We have some data buffered. Write it first.
       return other.stream->write(recvData.begin(), recvData.size())
-          .then([this, &other]() {
+          .then([this, &other, size = recvData.size()]() {
         recvData = nullptr;
+        sentBytes += size;
```

Comment:
I'm thinking I got this backwards. `a.pumpTo(b)` should only increment `receivedBytes` on `a`. Eventually we'll call `b.send()` which will increment `sentBytes` on `b`. Is that right?

Reply:
On this line specifically, I think you want to increment `other.sentBytes`. I think that `this->receivedBytes` has already accounted for bytes in the `recvData` buffer.